cfgov/unprocessed/js/modules/jquery/cf_notifier.js

```diff
@@ -51,9 +49,13 @@ var _notifier = {
       state:   settings.state,
       icon:    icon[settings.state]
     };
-    var template = handlebars.compile( settings.template );
+    var template = settings.template.replace( /\{\{\s?(.*?)\s?\}\}/g,
+      function( match, token ) {
+        return data[token];
```

Comment:
Fixed in https://github.com/cfpb/cfgov-refresh/pull/2457/commits/db93ec3aa5aad85c58412a86cdba58db0bfbe7dc. Currently working on removing this code so I'm going to leave as-is.